<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTCPCom</class>
 <widget class="QMainWindow" name="MyTCPCom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTCPCom</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxLeft">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radioButtonLeft1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLeft1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLeft2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonLeft2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radioButtonRight2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRight1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRight2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRight1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>Operate</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionSet"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperate"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="text">
    <string>Set</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
